src/index.js

```diff
@@ -1,5 +1,10 @@
 import isFunction from 'lodash/isFunction';
 import isObject from 'lodash/isObject';
+import assign from 'lodash/assign';
```

Comment:
Because most bundlers do "tree shaking" by file names currently. But I'm fine to change it to `import _ from 'lodash';` because there is the babel plugin (which you showcased in stampit) which does lodash tree shaking. :)
I'll do that same change to the `compose.js` file too.